api/src/main/java/org/jenkinsci/plugins/workflow/graph/FlowNode.java

```diff
@@ -141,6 +141,20 @@ public String getDisplayName() {
         else            return getTypeDisplayName();
```

Comment:
Can we deprecate `getDisplayName`? If so, should the `@Exported` annotation be moved to another method (with `name="displayName"` added)?